entrypoint.sh

```diff
@@ -10,8 +13,10 @@ if [ "$(ls -A /src)" ]; then
     # Install packages from requirements files
     for r in /src/*.pip ; do
```

Comment:
```suggestion
    for r in /src/*/requirements/*.pip ; do
```

Reply:
Are you sure you want to restrict to `requirements`? You could have plugins following the default python convention with a `.pip` in the base folder.
How about both: `for r in /src/*/*.pip /src/*/requirements/*.pip; do`?

Reply:
I think `requirements ` is enough because this is [the recommended pattern](https://github.com/opendatateam/cookiecutter-udata-plugin/tree/master/%7B%7Bcookiecutter.project_slug%7D%7D) for udata plugins and even [external organizations](https://github.com/koumoul-dev/udata-data-fair) are following this convention.